src/bin/pg_dump/pg_dump.c

```diff
@@ -10521,6 +10521,9 @@ dumpExtension(Archive *fout, const ExtensionInfo *extinfo)
 
 	qextname = pg_strdup(fmtId(extinfo->dobj.name));
 
+	if (strstr(qextname, "babelfishpg_common") && isBabelfishDatabase(fout))
```

Comment:
I think we will only get this issue with tables and views, so we could set/reset this guc in `dumpTableSchema` function.

Reply:
As discussed, we will use dump_restore GUC in this case.